### PR TITLE
Update node-api with vote extrinsic.

### PR DESCRIPTION
**Pull request by benoitdevos**
logion-network/logion-internal#725

### CODE REVIEW

File: packages/crossmint/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@logion/crossmint",
-  "version": "0.1.6-2",
+  "version": "0.1.7-1",
```
Comment (gdethier) [inline packages/crossmint/package.json:3]: I think you could have just bumped the micro (no code change) but it's OK.

File: packages/client/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@logion/client",
-  "version": "0.18.0-5",
+  "version": "0.19.0-1",
```
Comment (gdethier) [inline packages/client/package.json:3]: I think you could have just bumped the micro (no code change) but it's OK.

File: packages/extension/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@logion/extension",
-  "version": "0.5.4-2",
+  "version": "0.5.5-1",
```
Comment (gdethier) [inline packages/extension/package.json:3]: I think you could have just bumped the micro (no code change) but it's OK.

File: packages/node-api/src/interfaces/augment-types.ts [hunk 4]
```diff
@@ -5,9 +5,6 @@
...
-import type { BeefyKey, FullIdentification, IdentificationTuple, Keys, MembershipProof, SessionIndex, SessionKeys1, SessionKeys10, SessionKeys10B, SessionKeys2, SessionKeys3, SessionKeys4, SessionKeys5, SessionKeys6, SessionKeys6B, SessionKeys7, SessionKeys7B, SessionKeys8, SessionKeys8B, SessionKeys9, SessionKeys9B, ValidatorCount } from '@logion/node-api/dist/interfaces/session';
```
Comment (gdethier) [inline packages/node-api/src/interfaces/augment-types.ts:10]: I think there is something broken here...

File: packages/node-api/src/interfaces/augment-types.ts [hunk 5]
```diff
@@ -41,7 +38,7 @@ import type { AssetOptions, Owner, PermissionLatest, PermissionVersions, Permiss
...
 import type { AuthorityIndex, AuthorityList, AuthoritySet, AuthoritySetChange, AuthoritySetChanges, AuthorityWeight, DelayKind, DelayKindBest, EncodedFinalityProofs, ForkTreePendingChange, ForkTreePendingChangeNode, GrandpaCommit, GrandpaEquivocation, GrandpaEquivocationProof, GrandpaEquivocationValue, GrandpaJustification, GrandpaPrecommit, GrandpaPrevote, GrandpaSignedPrecommit, JustificationNotification, KeyOwnerProof, NextAuthority, PendingChange, PendingPause, PendingResume, Precommits, Prevotes, ReportedRoundStates, RoundState, SetId, StoredPendingChange, StoredState } from '@polkadot/types/interfaces/grandpa';
 import type { IdentityFields, IdentityInfo, IdentityInfoAdditional, IdentityInfoTo198, IdentityJudgement, RegistrarIndex, RegistrarInfo, Registration, RegistrationJudgement, RegistrationTo198 } from '@polkadot/types/interfaces/identity';
-import type { AuthIndex, AuthoritySignature, Heartbeat, HeartbeatTo244, OpaqueMultiaddr, OpaqueNetworkState } from '@polkadot/types/interfaces/imOnline';
+import type { AuthIndex, AuthoritySignature, Heartbeat, HeartbeatTo244, OpaqueMultiaddr, OpaqueNetworkState, OpaquePeerId } from '@polkadot/types/interfaces/imOnline';
```
Comment (gdethier) [inline packages/node-api/src/interfaces/augment-types.ts:41]: This is potentially a problem.

File: packages/node-api/src/interfaces/augment-types.ts [hunk 6]
```diff
@@ -230,11 +229,6 @@ declare module '@polkadot/types/types/registry' {
     CollationInfoV1: CollationInfoV1;
     CollatorId: CollatorId;
     CollatorSignature: CollatorSignature;
-    CollectionItem: CollectionItem;
-    CollectionItemFile: CollectionItemFile;
-    CollectionItemId: CollectionItemId;
-    CollectionItemToken: CollectionItemToken;
-    CollectionSize: CollectionSize;
```
Comment (gdethier) [inline packages/node-api/src/interfaces/augment-types.ts:237]: This is not normal.

**Review by gdethier — CHANGES_REQUESTED** [review]
I think something is broken in the generation code. Let's discuss.

File: packages/node-api/src/interfaces/augment-types.ts (hunk 5 above)
Comment (benoitdevos) [reply packages/node-api/src/interfaces/augment-types.ts]: Fixed

File: packages/node-api/src/interfaces/augment-types.ts (hunk 6 above)
Comment (benoitdevos) [reply packages/node-api/src/interfaces/augment-types.ts]: Manually fixed.

File: packages/node-api/src/interfaces/augment-types.ts [hunk 7]
```diff
@@ -5,9 +5,7 @@
...
-import type { BeefyKey, FullIdentification, IdentificationTuple, Keys, MembershipProof, SessionIndex, SessionKeys1, SessionKeys10, SessionKeys10B, SessionKeys2, SessionKeys3, SessionKeys4, SessionKeys5, SessionKeys6, SessionKeys6B, SessionKeys7, SessionKeys7B, SessionKeys8, SessionKeys8B, SessionKeys9, SessionKeys9B, ValidatorCount } from '@logion/node-api/dist/interfaces/session';
+import type { CollectionItem, CollectionItemFile, CollectionItemId, CollectionItemToken, CollectionSize, File, HostData, LegalOfficerCaseOf, LegalOfficerData, LoAuthorityListStorageVersion, LocId, LocLink, LocType, LocVoidInfo, MetadataItem, OpaquePeerId, StorageVersion, TermsAndConditionsElement } from '@logion/node-api/dist/interfaces/default';
```
Comment (gdethier) [inline packages/node-api/src/interfaces/augment-types.ts:8]: Default type definitions are back, but we are still lacking the overrides for runtime and session. See removed imports.

File: packages/node-api/src/interfaces/augment-types.ts [hunk 8]
```diff
@@ -877,7 +877,6 @@ declare module '@polkadot/types/types/registry' {
     ReportedRoundStates: ReportedRoundStates;
     Reporter: Reporter;
     ReportIdOf: ReportIdOf;
-    Requester: Requester;
```
Comment (gdethier) [inline packages/node-api/src/interfaces/augment-types.ts:880]: Maybe this has no impact but I don't see why it should be removed.

**Review by gdethier — CHANGES_REQUESTED** [review]
There are still some missing overrides.

**Review by gdethier — APPROVED** [review]
:+1:

**Pull request merged**